index.html

```diff
@@ -3543,6 +3543,12 @@ <h3>Algorithm</h3>
           </ol>
         </li>
         <li>If <var>compact IRI</var> is not <code>null</code>, return <var>compact IRI</var>.</li>
+        <li class="changed">To ensure that the <a>absolute IRI</a> <var>var</var> is
+          not confused with a <a>compact IRI</a>,
+          if the <a data-cite="RFC3986#section-3.1">IRI scheme</a> of <var>var</var>
+          matches any term in <a>active context</a>,
+          an <a data-link-for="JsonLdErrorCode">iri confused with prefix</a> error has been detected,
```

Comment:
Match test and capitalization in other codes.
```suggestion
          an <a data-link-for="JsonLdErrorCode">IRI confused with prefix</a> error has been detected,
```

Reply:
Did through separate commit, as there was a third.